build.sbt

```diff
@@ -1376,7 +1376,9 @@ lazy val jdkhttpServer: ProjectMatrix = (projectMatrix in file("server/jdkhttp-s
   .settings(commonJvmSettings)
   .settings(
     name := "tapir-jdkhttp-server",
-    libraryDependencies ++= loggerDependencies
+    libraryDependencies ++= Seq(
+      "org.apache.httpcomponents" % "httpmime" % "4.5.14"
```

Comment:
I saw finatra-server also depended on this, and it seems the `...ToResponseBody` part could be similar so i copied it.

Thoughts? Can we live with this, or do you want me to explore building the responsebody without a dependency?

Reply:
if it's needed, that's fine; also no problem with copying code across interpreters

however, there's a simple "mime database" here: https://github.com/softwaremill/tapir/blob/bfba44fa610656f542c36d0a979ee53cbfd2dad2/core/src/main/scalajvm/sttp/tapir/internal/MimeByExtensionDB.scala#L7, maybe it will be useful